src/styles/constants/_layout.scss

```diff
@@ -11,6 +11,9 @@ $bdl-border-radius-size-med: $bdl-border-radius-size * 1.5;
 $bdl-border-radius-size-large: $bdl-border-radius-size * 2;
 $bdl-border-radius-size-xlarge: $bdl-border-radius-size * 3;
 
+// Drop shadows
+$bdl-box-shadow-header: 0 0 10px 0 rgba(0, 0, 0, .05);
```

Comment:
Should this name be `$bdl-header-box-shadow` and `bdlHeaderBoxShadow`?

It seems to be the pattern used for buttons:
https://github.com/box/box-ui-elements/blob/master/src/styles/variables.js#L97-L102